Frontend/android-native/mobility-common/src/main/java/in/juspay/mobility/common/MobilityCommonBridge.java

```diff
@@ -772,7 +829,11 @@ protected void updateLastKnownLocation(String callback, boolean animate, final S
                             if (animate && googleMap != null) {
                                 LatLng latLng = new LatLng(lat, lng);
                                 if (userPositionMarker == null) {
-                                    upsertMarker(CURRENT_LOCATION, String.valueOf(lat), String.valueOf(lng), 160, 0.5f, 0.9f); //TODO this function will be removed
+                                    String marker = "";
+                                    if (!editLocation) {
```

Comment:
can we pass this as function parameter itself whether we want to add marker or not.